imagine-dragons-web/resources/css/home-stylesheet.css

```diff
@@ -0,0 +1,5 @@
+#home-img {
+    padding-top: 24%;
```

Comment:
using percentages, especially with padding is not good. why?

Reply:
because precentages is relative to the same property of the parent element - and the parent element (or the property I use - in my case is the padding) can be replaced easily, and in this case, the same element's property would change vary different from what I want it to be (especially with padding when there is not a real connection between the padding of the parent to the child's padding)